pkgs/development/python-modules/lightgbm/default.nix

```diff
@@ -8,8 +8,14 @@
 , scikit-learn
 , llvmPackages ? null
 , pythonOlder
+, python
+, ocl-icd
+, opencl-headers
+, boost
+, gpuSupport ? true
 }:
 
+with lib;
```

Comment:
```suggestion
```

Reply:
Please do not add new with's over the entire file as we are actively removing them from ``pkgs/``.

Reply:
Is there any RFC, or mention of this decision for a treewide style change anywhere?

Reply:
https://github.com/NixOS/nixpkgs/issues/208242

Reply:
Thanks. The PR has already been updated.

Reply:
it's okay to use it in meta FYI

Reply:
Should I update this PR, or is it fine?